userland/libtock/crt1.c

```diff
@@ -19,8 +19,15 @@ __attribute__ ((naked))
 void _start(void* mem_start __attribute__((unused)),
             void* app_heap_break __attribute__((unused)),
             void* kernel_memory_break __attribute__((unused))) {
-  asm volatile ("mov r9, sp");
-  main();
-  while (1) { yield(); }
+  /* 
+   * 1. Setup r9 to point to the GOT (assumes the kernel places the GOT right
+   *    above the stack)
+   * 2. Call `main`
+   * 3. Loop on `yield` forever
+   */
+  asm volatile ("mov r9, sp; \
+                 bl main; \
+                 1: bl yield; \
+                 b 1b");
```

Comment:
It means "only look back for labels of that name", which is important because `1:` is a local label, meaning it doesn't clobber the global namespace.